### PR TITLE
[Remove Stream/Optional usage] LPS-175370 module export-import-test

### PR DESCRIPTION
**Pull request by JanisZhang**
https://issues.liferay.com/browse/LPS-175370

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:export-import

**Comment by catarineaguiar** [comment]
ci:test:export-import

**Comment by catarineaguiar** [comment]
no regressions

File: modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/content/processor/test/DefaultExportImportContentProcessorTest.java [hunk 1]
```diff
@@ -877,13 +876,13 @@ private void _assertContainsBinary(
 		Pattern pattern = Pattern.compile(
 			String.format("/%s/%d/\\d+\\.\\d+$", className, classPK));
 
-		Stream<String> entriesStream = entries.stream();
+		entries = ListUtil.filter(entries, pattern.asPredicate());
```
Comment (dantewang) [inline modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/content/processor/test/DefaultExportImportContentProcessorTest.java:879]: @JanisZhang This looks nice in tests. Could you check if you can apply the pattern to https://github.com/liferay-core-infra/liferay-portal/pull/3662 also?

File: modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/content/processor/test/DefaultExportImportContentProcessorTest.java [hunk 2]
```diff
@@ -877,13 +876,13 @@ private void _assertContainsBinary(
...
 
-		Assert.assertTrue(
+		Assert.assertFalse(
 			String.format(
 				"%s does not contain a binary entry for %s with primary key %s",
 				entries.toString(), className, classPK),
-			entriesStream.anyMatch(pattern.asPredicate()));
+			entries.isEmpty());
```
Comment (dantewang) [inline modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/content/processor/test/DefaultExportImportContentProcessorTest.java:885]: @JanisZhang You can inline these to something like `Assert.assertTrue(..., ListUtil.isNotEmpty(ListUtil.filter(entries, predicate)))`.

File: modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/ExportedMissingReferenceBackwardCompatbilityExportImportTest.java [hunk 3]
```diff
@@ -143,12 +143,12 @@ public void testBackwardCompatibility() throws Exception {
...
 			@Override
 			public void evaluate() throws Throwable {
-				Stream<Method> methodStream = _parentTestMethods.stream();
+				List<Method> methods = ListUtil.filter(
+					_parentTestMethods,
+					m -> Objects.equals(
+						m.getName(), description.getMethodName()));
 
-				Assume.assumeTrue(
-					methodStream.noneMatch(
```
Comment (dantewang) [inline modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/ExportedMissingReferenceBackwardCompatbilityExportImportTest.java:149]: @JanisZhang In https://github.com/liferay-core-infra/liferay-portal/pull/3662, you converted some usages to anyMatch; one of them should be suitable for `noneMatch`. Please also apply to that.

File: modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/ExportedMissingReferenceBackwardCompatbilityExportImportTest.java [hunk 4]
```diff
@@ -143,12 +143,12 @@ public void testBackwardCompatibility() throws Exception {
...
-						m -> Objects.equals(
-							m.getName(), description.getMethodName())));
+				Assume.assumeTrue(methods.isEmpty());
```
Comment (dantewang) [inline modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/ExportedMissingReferenceBackwardCompatbilityExportImportTest.java:151]: @JanisZhang You can also inline here.

**Review by dantewang — CHANGES_REQUESTED** [review]
@JanisZhang Please see my comment, modify this PR as well as https://github.com/liferay-core-infra/liferay-portal/pull/3662, and resend, thanks!

**Pull request closed without merging**